
FAIL history.pushState() fires the navigate event assert_false: expected false got true

